docsrc/contributors/fx_converters.rst

```diff
@@ -0,0 +1,195 @@
+.. _conversion:
+
+FX Converters
+==================
+The converter library in Torch-TensorRT is located in ``TensorRT/py/torch_tensorrt/fx/converters``.
+They are categorized into - ``aten_ops_converters``, ``acc_ops_converters`` and ``nn_ops_converters``.
+The individual converters present are useful for the quantization workflow.
+The converters are registered using the ``tensorrt_converter``.
+
+Steps
+==================
+
+Operation Sets
+-------------------
+There are three different converter sets for FX in torch_tensorrt. Depending on whether the operation is generated using acc_trace, aten_trace or fx_trace, the converters are categorized to one of the three operation sets - 
+``aten_ops_converters``, ``acc_ops_converters`` or ``nn_ops_converters``.  The converters are registered using ``tensorrt_converter`` decorator. The function decorated
+has the arguments - ``network, target, args, kwargs, name``,  which is common across all the operators schema.
+These functions are mapped in the ``aten`` converter registry dictionary (at present a compilation of FX and dynamo converters, FX will be deprecated soon), with key as the function target name.
+    
+    * acc_ops_converters
+        *  acc_trace is produced by ``torch_tensorrt.fx.tracer.acc_tracer.acc_tracer.trace``.
+    * aten_ops
+        There are two options at present for this
+            #.   Dynamo: aten_trace is produced by ``torch_tensorrt.dynamo.backend.compile``. The second round of trace is produced by ``aot_torch_tensorrt_aten_backend`` by invoking ``aot_module_simplified`` from ``torch._functorch.aot_autograd``
+            #.   FX: aten_trace is produced by ``torch_tensorrt.fx.tracer.dispatch_tracer.aten_tracer.trace``. This flow is more common currently, but this will soon be deprecated in torch_tensorrt.
```

Comment:
Talk to @peri044 about the new workflow for dynamo.compile

Reply:
@peri044 could you mention some points here?